bookshelf.js

```diff
@@ -39,4 +40,16 @@ spines.map(function (s, i) {
   covers[i].style.top = `${280 - randomHeight}px`;
 
   tops[i].style.top = `${280 - randomHeight}px`;
+
+  // 75 characters above spoils the alignment
+  if (titles[i].innerHTML.length > 75) {
+    titles[i].innerHTML = titles[i].innerHTML.substring(0, 73) + "...";
+  }
```

Comment:
Isn't this more elegantly achieved using `text-overflow: ellipsis;` (and possibly some other CSS properties) ?

Reply:
`text-overflow: ellipsis` is a good property but I could not get it to work even with other properties. However I found this in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow)
> The text-overflow property only affects content that is overflowing a block container element in its inline progression direction (not text overflowing at the bottom of a box, for example).

Reply:
Also just a question - What is the problem with JavaScript? (A generalized question for both the comments)

Reply:
No problem with JS, I think it's more about just using the right tool for the job. 

I'd like to have a go at seeing if `text-overflow`could work, hope that's ok. Otherwise this should be an okay way to deal with it. I appreciate the help 👍

Reply:
Hey, I just had a go at trying to fix this and yeah, it's very difficult to get it perfect using just CSS. I got close but not quite. The issue is dealing not just with long titles but specifically long _words_. 

Anyway, your solution works well, however your chosen values didn't work for me. I had to reduce the character length to `40` in the JS. I experimented with various different titles with long words, many short words, etc... Are you able to confirm?

Reply:
40 instead fits perfectly for me and my chosen values goes out of bounds
Browser - Firefox v100

Reply:
![image](https://user-images.githubusercontent.com/22571613/168487462-6d98a431-cb38-4a6d-a45c-7d9f4cec0c60.png)